### PR TITLE
status: Add 2024q2 FreeBSD Foundation report entry

### PR DESCRIPTION
**Pull request by Jehops**
Section authors:
- Introduction: Deb Goodkin <deb@freebsdfoundation.org>
- OS Improvements: Joe Mingrone <jrm@freebsdfoundation.org>
- Partnerships and Research: Greg Wallace <greg@freebsdfoundation.org>
- Advocacy: Anne Dickison <anne@freebsdfoundation.org>

### CODE REVIEW

File: website/content/en/status/report-2024-04-2024-06/freebsd-foundation.adoc [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+The Foundation is sponsoring 13 projects.
+
+* mailto:christos@FreeBSD.org[Christos Margiolis] continued to improve FreeBSD's audio stack and provide audio developers with useful tools and frameworks to facilitate sound development on FreeBSD.
+Refer to the <<_audio_stack_improvements,Audio Stack Improvements>> entry for details.
+
+* mailto:pjd@FreeBSD.org[Pawel Dawidek] is in the final stages of a project to add hierarchical rate limits to OpenZFS.
+The configured limits for bandwidth and operations will work like quotas.
+If a child dataset is configured with a higher limit than its parent, the parent's lower limit will be enforced.
+For details, refer to Pawel's link:https://github.com/openzfs/zfs/pull/16205[pull request in the OpenZFS repository].
```
Comment (pauamma) [inline website/content/en/status/report-2024-04-2024-06/freebsd-foundation.adoc:59]: Noting here that pjd@ submitted a report that still has to be committed. When it is, do you mind if we link to it from here, perhaps using "a project to add hierarchical rate limits to OpenZFS" as the link text?

Comment (Jehops) [reply website/content/en/status/report-2024-04-2024-06/freebsd-foundation.adoc:59]: Sounds good.  Thanks.

**Review by pauamma — APPROVED** [review]

File: website/content/en/status/report-2024-04-2024-06/freebsd-foundation.adoc (hunk 1 above)
Comment (Jehops) [reply website/content/en/status/report-2024-04-2024-06/freebsd-foundation.adoc:59]: @pauamma, could you share the title of Pawe's report entry?  That way, I can try to save you a bit of work.

Comment (pauamma) [reply website/content/en/status/report-2024-04-2024-06/freebsd-foundation.adoc:59]: "Hierarchical rate limits for OpenZFS"

**Review by lsalvadore — APPROVED** [review]
Approved once the suggestion from @pauamma is integrated into the report. It would also be great if you could remove the contractions from the first part. Thanks

**Pull request merged**